src/snowflake/cli/_plugins/nativeapp/application_package_entity.py

```diff
@@ -723,6 +748,116 @@ def check_index_changes_in_git_repo(
         except InvalidGitRepositoryError:
             pass  # not a git repository, which is acceptable
 
+    @classmethod
+    def version_drop(
+        cls,
+        console: AbstractConsole,
+        project_root: Path,
+        deploy_root: Path,
+        bundle_root: Path,
+        generated_root: Path,
+        artifacts: list[PathMapping],
+        package_name: str,
+        package_role: str,
+        package_distribution: str,
+        version: Optional[str],
+        force: bool,
+        interactive: bool,
+    ):
+        """
+        Drops a version defined in an application package. If --force is provided, then no user prompts will be executed.
+        """
+        is_interactive = False
```

Comment:
it might be confusing to have 2 similar variables. Can we do something like this instead?
```
if force:
  interactive = False
  policy = AllowAlwaysPolicy()
else:
  policy = AskAlwaysPolicy() if interactive else DenyAlwaysPolicy()
```

Reply:
this was copied verbatim from https://github.com/snowflakedb/snowflake-cli/pull/1610/files#diff-7f950152d04e5fc16d6529457263e2b23df2e9f6a0023f6572d49bdb5a9a9d38L140-L147 but yeah i can change it. I'm planning a refactor of this force/interactive/policy stuff anyways